packages/did-core-test-server/suites/implementations/universal-resolver-dereferencer-tests.json

```diff
@@ -0,0 +1,155 @@
+{
+    "implementation" : "Universal Resolver",
+    "implementer" : "Decentralized Identity Foundation and Contributors",
+    "didMethod" : "did:sov",
+    "expectedOutcomes" : {
+        "defaultOutcome" : [ 0, 1 ]
+    },
+    "executions" : [ {
+        "function" : "dereference",
+        "input" : {
+            "didUrl" : "did:sov:WRfXPg8dantKVubE3HX8pw",
+            "dereferenceOptions" : {
+                "accept" : "application/did+ld+json"
+            }
+        },
+        "output" : {
+            "dereferencingMetadata" : {
+                "contentType" : "application/did+ld+json"
+            },
+            "contentStream" : "{\"@context\":[\"https://www.w3.org/ns/did/v1\"],\"id\":\"did:sov:WRfXPg8dantKVubE3HX8pw\",\"verificationMethod\":[{\"type\":\"Ed25519VerificationKey2018\",\"id\":\"did:sov:WRfXPg8dantKVubE3HX8pw#key-1\",\"publicKeyBase58\":\"H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV\"}],\"authentication\":[{\"type\":\"Ed25519VerificationKey2018\",\"id\":\"did:sov:WRfXPg8dantKVubE3HX8pw#key-1\",\"publicKeyBase58\":\"H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV\"}],\"assertionMethod\":[{\"type\":\"Ed25519VerificationKey2018\",\"id\":\"did:sov:WRfXPg8dantKVubE3HX8pw#key-1\",\"publicKeyBase58\":\"H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV\"}],\"service\":[{\"type\":\"agent\",\"serviceEndpoint\":\"https://agents.danubeclouds.com/agent/WRfXPg8dantKVubE3HX8pw\"},{\"type\":\"xdi\",\"serviceEndpoint\":\"https://xdi03-at.danubeclouds.com/cl/+!:did:sov:WRfXPg8dantKVubE3HX8pw\"}]}",
```

Comment:
looks like your context is not correct, and does not define "Ed25519VerificationKey2018"...

Does this indicate a problem in the dereference tests? or the did method?

Reply:
I reviewed the section: https://w3c.github.io/did-core/#production-0

> A conforming producer that generates a JSON-LD representation SHOULD NOT produce a DID document that contains terms not defined via the @context as conforming consumers are expected to remove unknown terms.

Seems like your example is legal.

Reply:
@OR13 You're right of course that this DID document has undefined JSON-LD terms, and that this needs to be fixed. This is also true for many other current test reports in the repo.

In our case, the specific Universal Resolver driver needs to be updated. We'll try to fix as many of the Universal Resolver drivers as we can.

However, strictly speaking, this doesn't violate any of the normative MUST statements in the DID Core specification, therefore the test suite passes....

Reply:
@peacekeeper yep, I agree.